Boardless/Assets/Scripts/ObjectGenerator.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObjectGenerator : MonoBehaviour
+{
+    // The object to generate.
+    public GameObject obj;
+
+    // The position of the generated object.
+    public Vector3 pos; 
+
+    // Generates the preset game object at the given position.
+    public void generate()
```

Comment:
public members should capitalize the first letter per coding convention: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#pascal-case